src/dcommands/names.js

```diff
@@ -39,62 +39,41 @@ class NamesCommand extends BaseCommand {
+                if ((response && bu.parseBoolean(response.content)) || input.y) {
+                    await r.table('user').get(user.id).update({usernames : filteredUsers}).run();
+                    await bu.send(msg, `Succesfully removed ${storedUser.usernames.length - filteredUsers.length} username${storedUser.usernames.length - filteredUsers.length > 1 ? 's' : ''}.`);
                 } else {
-                    bu.send(msg, `Removal cancelled.`)
+                    await bu.send(msg, `OK, not removing your usernames!`)
                 }
                 if (prompt.prompt)
                     await bot.deleteMessage(prompt.prompt.channel.id, prompt.prompt.id);
                 return;
-            } else if (input.r.length == 0) {
-                //lookup
+            } else if (input.r.length === 0) {
                 let matches = storedUser.usernames.map((u, i) => { return { content: u.name, value: i } });
                 let lookup = await bu.createLookup(msg, 'username', matches);
-                if (await lookup == null)
+                if (!lookup)
                     return;
                 let removedUsername = storedUser.usernames.splice(lookup, 1)[0];
                 await r.table('user').get(user.id).update(storedUser).run();
                 return bu.send(msg, `Succesfully removed **${removedUsername}**!`);
-            } else if (input.r.length > 0) {
-                //filter
-                let name = input.r.join(' ');
-                let filteredUsers = storedUser.usernames.filter(u => !(u.name.toLowerCase().includes(name)));
-                if (filteredUsers.length == storedUser.usernames.length)
-                    return bu.send(msg, `No usernames found!`);
-                let prompt, response;
-
-                if (!input.y) {
-                    prompt = await bu.createPrompt(msg, `Are you sure you want to remove ${storedUser.usernames.length - filteredUsers.length} username${storedUser.usernames.length - filteredUsers.length > 1 ? 's' : ''}?` +
-                        `\nType \`yes\` or anything else to cancel`, null, 60000);
-                    response = await prompt.response || {};
-                }
-                if (!response || bu.parseBoolean(response.content)) {
-                    await r.table('user').get(user.id).update({ usernames: filteredUsers }).run();
-                    await bu.send(msg, `Succesfully removed ${storedUser.usernames.length - filteredUsers.length} username${storedUser.usernames.length - filteredUsers.length > 1 ? 's' : ''}.`);
-                } else {
-                    await bu.send(msg, `OK, not removing your usernames!`);
-                }
-                if (response)
-                    await bot.deleteMessage(prompt.prompt.channel.id, prompt.prompt.id);
-                return;
-            }
-
-
-        }
+            };
+        };
```

Comment:
why do you add semi columns ?

Reply:
That might be because I use a code formatter/prettifier :/